### PR TITLE
Generate uscore and aubase with latest tool changes

### PR DESCRIPTION
**Pull request by ThishaniLucas**

### CODE REVIEW

**Review by nirmal070125 — DISMISSED** [review]

File: aubase410/Module.md [hunk 1]
```diff
@@ -43,9 +44,9 @@ function parseSamplePatient() returns aubase410:AUBasePatient {
     };
 
     do {
-        anydata parsedResult = check r4:parse(patientPayload, aubase410:AUBasePatient);
```
Comment (AnjanaSenanayake) [inline aubase410/Module.md:46]: Are these genereted in this manner..this should have the r4 import given parse is fron r4?

File: aubase410/Module.md [hunk 2]
```diff
@@ -1,27 +1,28 @@
...
 
-1. Install Ballerina 2201.5.0 (Swan Lake Update 5)
+1. Install Ballerina 2201.6.0 or later
 
 ### 1. Parse JSON FHIR resource to FHIR resource model
-Sample below is using the AUBasePatient resource in `health.fhir.r4.aubase410` package.
+Sample below is using the Patient resource in `health.fhir.r4.aubase410` package.
 
 ```ballerina
-import ballerinax/health.fhir.r4;
```
Comment (AnjanaSenanayake) [inline aubase410/Module.md:15]: the r4 import should be there to work this sample?

File: aubase410/Package.md [hunk 3]
```diff
@@ -53,37 +52,38 @@ This package includes, FHIR AUBase Resource types.
...
-[m13]: https://lib.ballerina.io/ballerinax/health.fhir.r4.aubase410/1.0.3/#AUBaseDiagnosticRequest
-[m14]: https://lib.ballerina.io/ballerinax/health.fhir.r4.aubase410/1.0.3/#AUBaseMedicationAdministration
-[m15]: https://lib.ballerina.io/ballerinax/health.fhir.r4.aubase410/1.0.3/#AUBaseMedicationStatement
-[m16]: https://lib.ballerina.io/ballerinax/health.fhir.r4.aubase410/1.0.3/#AUBaseSpecimen
-[m17]: https://lib.ballerina.io/ballerinax/health.fhir.r4.aubase410/1.0.3/#AUBaseBodyStructure
-[m18]: https://lib.ballerina.io/ballerinax/health.fhir.r4.aubase410/1.0.3/#AUBaseMedicationDispense
-[m19]: https://lib.ballerina.io/ballerinax/health.fhir.r4.aubase410/1.0.3/#AUBaseCondition
-[m20]: https://lib.ballerina.io/ballerinax/health.fhir.r4.aubase410/1.0.3/#AUBaseDiagnosticImagingReport
-[m21]: https://lib.ballerina.io/ballerinax/health.fhir.r4.aubase410/1.0.3/#AUHealthProgramParticipationSummary
-[m22]: https://lib.ballerina.io/ballerinax/health.fhir.r4.aubase410/1.0.3/#AUBaseRelatedPerson
```
Comment (AnjanaSenanayake) [inline aubase410/Package.md:77]: Are these manually edited? The new links looks incorrect

File: aubase410/Module.md (hunk 1 above)
Comment (ThishaniLucas) [reply aubase410/Module.md:46]: fixed

File: aubase410/Module.md (hunk 2 above)
Comment (ThishaniLucas) [reply aubase410/Module.md:15]: fixed

File: aubase410/Package.md (hunk 3 above)
Comment (ThishaniLucas) [reply aubase410/Package.md:77]: fixed

**Review by AnjanaSenanayake — APPROVED** [review]

**Pull request merged**